build-scripts/hack/util.py

```diff
@@ -48,3 +49,26 @@ def read_file(path: Path) -> str:
 
 def read_url(url: str) -> str:
     return urlopen(url).read().decode().strip()
+
+
+def helm_pull(chart_name: str, repo_url: str, version: str, destination: Path) -> None:
+    parse_output(
+        [
+            "helm",
+            "pull",
+            f"{repo_url}",
+            f"{chart_name}/{chart_name}",
```

Comment:
I think this won't work for example for contour it's: 
`sudo k8s helm pull oci://registry-1.docker.io/bitnamicharts/contour bitnami/contour --version 17.0.4 --destination ./k8s/manifests/charts/`
So something like f"{chart_maintainer}/{chart_name}", would be better.